src/main.js

```diff
@@ -5,13 +5,15 @@ var ideaForm = document.querySelector('.user-idea-form')
 var savedCardsGrid = document.querySelector('.saved-cards-grid')
 var userCardTitle = document.getElementById('userCardTitle');
 var userCardBody = document.getElementById('userCardBody');
-// var deleteIdeaCard = document.querySelector('.delete-card-x');
+var starImageSrc = document.querySelector('.favorited-star');
 
 ideaForm.addEventListener('keyup', checkUserInputs);
 saveIdeaBtn.addEventListener('click', createNewIdea);
-savedCardsGrid.addEventListener('click', deleteCard);
+savedCardsGrid.addEventListener('click', handleIdeaCardClick); //one function that would figure out what was clicked and then send you on to another fx
```

Comment:
I love how when we delete or favorite a card it is all handled by the same event listener!  Awesomeness!